sigh/lib/assets/resign.sh

```diff
@@ -106,37 +130,49 @@ TEMP_DIR="_floatsignTemp"
 NESTED_APP_REFERENCE_KEYS=(":WKCompanionAppBundleIdentifier" ":NSExtension:NSExtensionAttributes:WKAppBundleIdentifier")
 
 # options start index
-OPTIND=3
-while getopts p:d:e:k:b:n:v opt; do
-    case $opt in
-        p)
-            RAW_PROVISIONS+=("$OPTARG")
+shift 2
+
+# Parse args
+while [ "$1" != "" ]; do
+    case $1 in
+        -p | --provisioning )
+            shift
+            RAW_PROVISIONS+=("$1")
             ;;
-        d)
-            DISPLAY_NAME="$OPTARG"
+        -e | --entitlements )
+            shift
+            ENTITLEMENTS="$1"
             ;;
-        e)
-            ENTITLEMENTS="$OPTARG"
+        -d | --display-name )
+            shift
+            DISPLAY_NAME="$1"
             ;;
-        b)
-            BUNDLE_IDENTIFIER="$OPTARG"
+        -b | --bundle-id )
+            shift
+            BUNDLE_IDENTIFIER="$1"
             ;;
-        k)
-            KEYCHAIN="$OPTARG"
+        -k | --keychain )
+            shift
+            KEYCHAIN="$1"
             ;;
-        n)
-            VERSION_NUMBER="$OPTARG"
+        -n | --version-number )
+            shift
+            VERSION_NUMBER="$1"
             ;;
-        v)
+        -v | --verbose )
             VERBOSE="--verbose"
             ;;
-        \?)
-            error "Invalid option: -$OPTARG"
+        -h | --help )
+            usage
```

Comment:
@asfalcone Please don't merge this one yet. I just noticed there's no `usage` method declared, I will add one.